### ISSUE TITLE
“Smart” date detection/handling while importing activities

### ISSUE THREAD

**Issue by gjmveloso** [afadil/wealthfolio#172]
I’m just exploring this app for managing my investments and I realized that Activities Import from CSV are not managing `dd/MM/yyyy` format properly.

I would suggest exploring “smart” date parsing to better accommodate non-US users that may use different date format — or those that are just using foreign brokers/banks that provide CSV files with different localization settings from the operating system.

**Comment by Jubunt** [comment]
Good idea would also to explain the csv format required as it is not easy to understand why the import has gone wrong.
It took me a while to figure out that the wealthfolio is expecting a specific timestamp.
thanks

**Comment by richard-lee-863** [comment]
Would love a system setting to define date format or Locale to deal with this. Should be standard on all apps to assume users are internationally located, not just US users...

**Comment by afadil** [comment]
Fixed in v3, the app will try to detect the date format, and you can also specify a format.
